cpp/include/cudf/column/column.hpp

```diff
@@ -75,6 +76,21 @@ class column {
    */
   column(column&& other) noexcept;
 
+  /**
+   * @brief Move the contents from `other` to create a new column.
```

Comment:
```suggestion
   * @brief Construct a new column object from a moved `device_uvector`.
   *
   * This takes ownership of the `device_uvector`'s underlying `device_buffer`.
```

Reply:
"Construct a new column by taking ownership of the contents of a `device_uvector`.